src/core/tests/type_prop/scatter_elements_update.cpp

```diff
@@ -97,83 +101,100 @@ TEST(type_prop, scatter_elements_update_default_ctor) {
     EXPECT_THAT(get_shape_labels(scatter->get_output_partial_shape(0)), Each(ov::no_label));
 }
 
-TEST(type_prop, scatter_elements_update_preserve_partial_values_and_labels_via_evaluates_bounds) {
-    const auto data = op::Constant::create(element::i64, Shape{4}, {2, 3, 15, 4});
-    const auto indices = op::Constant::create(element::i64, Shape{2}, {3, 0});
+TYPED_TEST_P(ScatterElementsUpdateTest,
+             scatter_elements_update_preserve_partial_values_and_labels_via_evaluates_bounds) {
+    const auto data = op::v0::Constant::create(element::i64, Shape{4}, {2, 3, 15, 4});
+    const auto indices = op::v0::Constant::create(element::i64, Shape{2}, {3, 0});
     auto updates_shape = PartialShape{{10, 20}, {3, 4}};
     set_shape_labels(updates_shape, 20);
-    const auto axis = make_shared<op::Constant>(element::i16, Shape{}, 0);
+    const auto axis = make_shared<op::v0::Constant>(element::i16, Shape{}, 0);
 
-    const auto shape_of_u = std::make_shared<op::ShapeOf>(std::make_shared<op::Parameter>(element::i64, updates_shape));
+    const auto shape_of_u =
+        std::make_shared<op::v0::ShapeOf>(std::make_shared<op::v0::Parameter>(element::i64, updates_shape));
     const auto scatter = make_shared<op::v3::ScatterElementsUpdate>(data, indices, shape_of_u, axis);
 
-    auto param = std::make_shared<op::Parameter>(element::f32, PartialShape{1});
+    auto param = std::make_shared<op::v0::Parameter>(element::f32, PartialShape{1});
```

Comment:
As this scatter operator is used in shape-of sup-graphs (in models) the bound and label evaluation should be implemented to accurate calculate shapes. As new operator  not implements these functions this test should fail. I think this evaluate  bounds and label should be provided for v12 also, as usage of this op can be similar as for previous version.

Reply:
Fixed by moving the evaluate call to the base class.